azkaban-common/src/test/java/azkaban/project/ProjectManagerTest.java

```diff
@@ -0,0 +1,162 @@
+package azkaban.project;
+
+import azkaban.user.User;
+import azkaban.utils.Props;
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import org.apache.commons.io.FileUtils;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+public class ProjectManagerTest {
+
+  private File workingDir;
+
+  @Before
+  public void setUp() throws Exception {
+    System.out.println("projectManagerTest: create temp dir");
+    synchronized (this) {
```

Comment:
Different threads may be creating and deleting the same directory simultaneously, causing some inconsistency issues.